group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupCoordinatorServiceTest.java

```diff
@@ -1899,8 +1900,19 @@ public void testCommitTransactionalOffsets() throws ExecutionException, Interrup
         assertEquals(response, future.get());
     }
 
-    @Test
-    public void testCommitTransactionalOffsetsWithWrappedError() throws ExecutionException, InterruptedException {
+    private static Stream<Arguments> testCommitTransactionalOffsetsWithWrappedErrorSource() {
+        return Stream.of(
+            Arguments.arguments(new NotEnoughReplicasException(), Errors.COORDINATOR_NOT_AVAILABLE.code()),
+            Arguments.arguments(new NetworkException(), Errors.COORDINATOR_LOAD_IN_PROGRESS.code())
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("testCommitTransactionalOffsetsWithWrappedErrorSource")
```

Comment:
nit: Would it be possible to use a CsvSource with Errors,Errors? I am not sure if CsvSource works with Enums though.

Reply:
That appears to work, thanks!